src/main/java/de/tum/in/www1/artemis/service/TextAssessmentService.java

```diff
@@ -92,7 +95,6 @@ public Result saveAssessment(Long resultId, List<Feedback> textAssessment, TextE
         result.getFeedbacks().clear();
         for (Feedback feedback : textAssessment) {
             feedback.setPositive(feedback.getCredits() >= 0);
-            feedback.setType(FeedbackType.MANUAL);
```

Comment:
Is it really ok that we don't set the type anymore for the single feedback items?

Reply:
The correct feedback type is already set by the client.